packages/lib-react-components/src/PrimaryButton/themes/green.js

```diff
@@ -22,6 +22,11 @@ const theme = {
       const color = normalizeColor('status-ok', props.theme)
       const gradientShade = getGradientShade(color)
 
+      // This shouldn't be necessary where we're manually setting the color with our own ternary,
+      // but style collision is happening
+      // so the theme color under color.light isn't being applied
+      // needs more investigation, as it seems to be a Grommet bug
+      // const fontColor = props.theme.dark ? color : 'white'
```

Comment:
Is this comment still necessary with `fontColor` commented out?

Reply:
Oops! Yes, this is now cruft after I fixed the accessibility issue.